subjects/java/checkpoints/universal-greeting/README.md

```diff
@@ -0,0 +1,40 @@
+## UniversalGreeting
+
+### Instructions
+
+In a file named `UniversalGreeting.java` write a function `greeting` that takes a String language as a parameter and returns a greeting message based on the language as follows:
+
+FR : Bonjour comment allez-vous
```

Comment:
I would add some formatting to avoid confusion about the spaces in the return string.

`FR`: `Bonjour comment allez-vous`
... 
or 
"FR": "Bonjour comment allez-vou"
...

Also, is there a reason why the sentences don't end up with a question mark? :)

Reply:
Looks like i forget to add question marks,
yeah indeed, the formatting is a must here to avoid confusion. I will add it thank you.